Functions.Templates/Templates/SqlOutputBinding-PowerShell/function.json

```diff
@@ -0,0 +1,27 @@
+{
+    "bindings": [
+      {
+        "authLevel": "function",
+        "type": "httpTrigger",
+        "direction": "in",
+        "name": "Request",
+        "methods": [
+          "get",
```

Comment:
output binding run.ps1 example uses the request body. remove the "get" method option, which would not have a request body